frontends/p4/tableKeyNames.cpp

```diff
@@ -36,15 +36,15 @@ class KeyNameGenerator : public Inspector {
 
     void postorder(const IR::Member* expression) override {
         cstring n = getName(expression->expr);
-        if (n.isNullOrEmpty())
+        if (!n)
             return;
         name.emplace(expression, n + "." + expression->member);
     }
```

Comment:
This can be written more concisely as

    if (cstring n = ::get(name, expression->expr))
        name.emplace(expression, n + "." + expressions->member);